skills/tts-mode/hooks/ttsmode/main.go

```
@@ -0,0 +1,212 @@
+		path := store.LogPath()
+		if info, err := os.Stat(path); err == nil && info.Size() > maxLogBytes {
+			// Start over rather than keep a tail. What matters when speech goes
+			// quiet is the most recent failure, and the newest entries land
+			// immediately after this.
+			_ = os.Truncate(path, 0)
+		}
+		file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o600)
+		if err != nil {
+			return
+		}
+		defer file.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `file.Close` is not checked

(errcheck)